doc/CONFIGURATION.md

```diff
@@ -304,7 +304,7 @@ const node = await Libp2p.create({
   },
   config: {
     peerDiscovery: {
-      webRTCStar: {
+      [WebRTCStar.tag]: {
```

Comment:
That is a good point! We are using the tag's for discovery configurations everywhere. I also saw the `WebRTCStar.prototype[Symbol.toStringTag]` and it is being used for the configuration of a transport.
I think we should clean this up in #576 , but now I think we should guarantee consistency between all the discovery services. For transport configuration, we should use the symbol property. While for webrtct, it will be the same, for other transport/discovery it will not be.